editor/src/document/properties_panel_message_handler.rs

```diff
@@ -208,9 +224,101 @@ impl MessageHandler<PropertiesPanelMessage, &GrapheneDocument> for PropertiesPan
 	}
 }
 
-fn register_layer_properties(layer: &Layer, responses: &mut VecDeque<Message>) {
+fn register_artboard_layer_properties(layer: &Layer, responses: &mut VecDeque<Message>) {
+	let options_bar = vec![LayoutRow::Row {
+		widgets: vec![
+			// TODO: Replace with custom Artboard Icon
+			WidgetHolder::new(Widget::IconLabel(IconLabel {
+				icon: "NodeShape".into(),
```

Comment:
@Keavon may want to create an icon for art boards

Reply:
@Keavon Is this something I should wait on before merging this PR?

Reply:
I'm working on it right now.